sdk/python/dev/dagger.json

```diff
@@ -1,12 +1,7 @@
 {
   "name": "python-sdk-dev",
   "sdk": "python",
-  "exclude": [
-    "**/.venv",
-    "**/__pycache__",
-    "**/.*cache",
-    "sdk"
-  ],
+  "exclude": ["**/.venv", "**/__pycache__", "**/.*cache"],
```

Comment:
It's a shame that the sdk exclude was removed here. There's a bug where include/exclude patterns from dependencies aren't prefixed with the relative paths to them.